egs/libri_css/s5_css/run.sh

```diff
@@ -0,0 +1,243 @@
+      data/${datadir} \
+      exp/${datadir}_diarization
+  done
+fi
+
+#######################################################################
+# Decode diarized output using trained chain model
+#######################################################################
+if [ $stage -le 4 ]; then
+  for datadir in ${test_sets}; do
+    asr_nj=$(wc -l < "data/$datadir/wav.scp")
+    nj=$(echo $((decode_nj>asr_nj ? asr_nj : decode_nj)))
+    local/decode_diarized_css.sh --nj $nj --cmd "$decode_cmd" --stage $decode_diarize_stage \
+      --lm-suffix "_tgsmall" --acwt 1.0 --post-decode-acwt 10.0 \
+      exp/${datadir}_diarization/rttm.post data/$datadir data/lang_test_tgsmall \
+      exp/chain${nnet3_affix}/tdnn_${affix} exp/nnet3${nnet3_affix} \
+      data/${datadir}_diarized || exit 1
+  done
+fi
+
+#######################################################################
+# Score decoded dev/eval sets (only if we are not rescoring)
+#######################################################################
+if [ $stage -le 5 ] && [ ! $rnnlm_rescore ]; then
+  # please specify both dev and eval set directories so that the search parameters
+  # (insertion penalty and language model weight) will be tuned using the dev set
+  local/score_reco_diarized.sh --stage $score_stage \
+      --multistream true \
+      --dev_decodedir exp/chain${nnet3_affix}/tdnn_${affix}/decode_${dev_set}_diarized_2stage \
+      --dev_datadir ${dev_set}_diarized_hires \
+      --eval_decodedir exp/chain${nnet3_affix}/tdnn_${affix}/decode_${eval_set}_diarized_2stage \
+      --eval_datadir ${eval_set}_diarized_hires
+fi
+
+############################################################################
+# RNNLM rescoring
+############################################################################
+if $rnnlm_rescore; then
+  if [ $stage -le 6 ]; then
+    echo "$0: Perform RNNLM lattice-rescoring"
+    pruned=
+    ac_model_dir=exp/chain${nnet3_affix}/tdnn_${affix}
+    if $pruned_rescore; then
+      pruned=_pruned
+    fi
+    for decode_set in $test_sets; do
+      decode_dir=${ac_model_dir}/decode_${decode_set}_diarized_2stage
+      # Lattice rescoring
+      rnnlm/lmrescore$pruned.sh \
+          --cmd "$decode_cmd --mem 8G" \
+          --weight 0.45 --max-ngram-order $ngram_order \
+          data/lang_test_tgsmall $rnnlm_dir \
+          data/${decode_set}_diarized_hires ${decode_dir} \
+          ${ac_model_dir}/decode_${decode_set}_diarized_2stage_rescore
+    done
+  fi
+  
+  if [ $stage -le 7 ]; then
+    echo "$0: WERs after rescoring with $rnnlm_dir"
+    local/score_reco_diarized.sh --stage $score_stage \
+        --multistream true \
+        --dev_decodedir exp/chain${nnet3_affix}/tdnn_${affix}/decode_${dev_set}_diarized_2stage_rescore \
+        --dev_datadir ${dev_set}_diarized_hires \
+        --eval_decodedir exp/chain${nnet3_affix}/tdnn_${affix}/decode_${eval_set}_diarized_2stage_rescore \
+        --eval_datadir ${eval_set}_diarized_hires
```

Comment:
Pass `--cmd` to this command.
```suggestion
    local/score_reco_diarized.sh --cmd "$train_cmd" --stage $score_stage \
        --multistream true \
        --dev_decodedir exp/chain${nnet3_affix}/tdnn_${affix}/decode_${dev_set}_diarized_2stage_rescore \
        --dev_datadir ${dev_set}_diarized_hires \
        --eval_decodedir exp/chain${nnet3_affix}/tdnn_${affix}/decode_${eval_set}_diarized_2stage_rescore \
        --eval_datadir ${eval_set}_diarized_hires
```